outputs/aws.go

```diff
@@ -189,3 +192,80 @@ func (c *Client) PublishTopic(falcopayload types.FalcoPayload) {
 	c.Stats.AWSSNS.Add(OK, 1)
 	c.PromStats.Outputs.With(map[string]string{"destination": "awssns", "status": OK}).Inc()
 }
+
+// SendCloudWatchLog sends a message to CloudWatch Log
+func (c *Client) SendCloudWatchLog(falcopayload types.FalcoPayload) {
+	svc := cloudwatchlogs.New(c.AWSSession)
+
+	f, _ := json.Marshal(falcopayload)
+
+	c.Stats.AWSCloudWatchLogs.Add(Total, 1)
+
+	if c.Config.AWS.CloudWatchLogs.LogStream == "" {
+		streamName := "falcosidekick-logstream"
+		log.Printf("[INFO]  : %v CloudWatchLogs - Log Stream not configured creating one called %s\n", c.OutputType, streamName)
+		inputLogStream := &cloudwatchlogs.CreateLogStreamInput{
+			LogGroupName:  aws.String(c.Config.AWS.CloudWatchLogs.LogGroup),
+			LogStreamName: aws.String(streamName),
+		}
+
+		_, err := svc.CreateLogStream(inputLogStream)
+		if err != nil {
+			if awsErr, ok := err.(awserr.Error); ok && awsErr.Code() == cloudwatchlogs.ErrCodeResourceAlreadyExistsException {
+				log.Printf("[INFO]  : %v CloudWatchLogs - Log Stream %s already exist, reusing...\n", c.OutputType, streamName)
```

Comment:
we'll get a log line for each event, this is an useless flood. can you move the check/creation of the stream when you create the client please?

Reply:
No, it will only log in case you restart the process and it will try to create the logstream (if the variable logstream is not set)
It will log the message just once in case the stream already exists.

Reply:
👍